compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/statements/MatchStatementTest.java

```diff
@@ -49,4 +54,9 @@ public void testMatchStmtBodyRecovery() {
     public void testMatchStmtRecovery() {
         testFile("match-stmt/match_stmt_source_04.bal", "match-stmt/match_stmt_assert_04.json");
     }
+
+    @Test
+    public void testMatchStmtRecoveryMissingVarKeyword() {
+        testFile("match-stmt/match_stmt_source_06.bal", "match-stmt/match_stmt_assert_06.json");
+    }
```

Comment:
Shall we add some more negative tests?
Like missing commas, additional commas, etc, inside a binding-pattern in a match-pattern..

Reply:
Okay

Reply:
Done